cmd/ftl/cmd_call.go

```diff
@@ -39,8 +41,43 @@ func (c *callCmd) Run(ctx context.Context, client ftlv1connect.VerbServiceClient
 	if err != nil {
 		return fmt.Errorf("invalid request: %w", err)
 	}
+
+	logger.Debugf("Calling %s", c.Verb)
+	// lookup the verbs
+	res, err := ctlCli.GetVerbs(ctx, connect.NewRequest(&ftlv1.VerbsRequest{}))
+	if err != nil {
+		return err
+	}
+
+	var foundVerb *reflection.Ref
+	suggestions := []string{}
+	verbs := res.Msg.GetVerbs()
+	logger.Debugf("Found %d verbs", len(verbs))
+	needle := []rune(fmt.Sprintf("%s.%s", c.Verb.Module, c.Verb.Name))
```

Comment:
I think this should work no?

```suggestion
	needle := []rune(c.Verb.Module, c.Verb.Name)
```

Reply:
Updated whoops, some weird undo issue.

Reply:
theres a "." missing, no?

Reply:
Ah sorry, this is a `schemapb.Ref`, you're correct - it is a shitty protobuf String() method